Makefile

```diff
@@ -106,7 +106,12 @@ test-cli:
 	rm -r test_cli
 	@echo ""
 
-test-js:
+mock-integration-docs:
+	@echo "--> Mocking platform json assets"
+	@mkdir -p ./src/sentry/integration-docs/
+	@if [ ! -f "./src/sentry/integration-docs/_platforms.json" ]; then echo "{\"platforms\": []}" >./src/sentry/integration-docs/_platforms.json; fi
```

Comment:
Why don't we just gracefully handle this inside of `integrationdocs.py` ? There's no reason to need to dump this stub file on disk from what I can tell.